examples/react/components/Content.tsx

```diff
@@ -248,12 +248,21 @@ const Content: React.FC = () => {
       signature: signedMessage.signature,
       callbackUrl: message.callbackUrl,
     });
+
+    if (!verifiedSignature) {
+      alert("Invalid Signature: Failed to verify the signature");
+    }
```

Comment:
Looks like the `message` is not being signed correctly, I added this check here just for testing and will be removed before merging:

![image](https://github.com/near/wallet-selector/assets/95851345/9131dc5a-9f3c-4522-b5ac-4095e2285ce5)

Reply:
Sender has fixed this issue, we're just waiting for the release on Chrome.

Reply:
Marking this as resolved after testing out the sender locally, we now are just waiting for Sender to be published.